frontend/cypress/.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules/
+node/
```

Comment:
Is this 2 necessary? The project root is already ignoring `node_modules` though?